bonded-roles/src/main/java/bisq/bonded_roles/BondedRolesService.java

```diff
@@ -64,10 +64,13 @@ public static Config from(com.typesafe.config.Config config) {
     private final DifficultyAdjustmentService difficultyAdjustmentService;
     private final ReleaseNotificationsService releaseNotificationsService;
 
-    public BondedRolesService(Config config, PersistenceService persistenceService, NetworkService networkService) {
+    public BondedRolesService(Config config, PersistenceService persistenceService, NetworkService networkService,
+                              boolean skipMarketPriceService) {
```

Comment:
I think a better way is to add an `enabled` boolean field to the `marketPrice` config node and use that in the MarketPriceService. This limits the change to that class only and we do not add config logic in ApplicationServices.